package-lock.json

```diff
@@ -24856,37 +26892,6 @@
         "react-dom": "*"
       }
     },
-    "packages/sui-react-head/node_modules/@types/aria-query": {
```

Comment:
Is this needed ?

Reply:
Let me check it running the phoenix script again

Reply:
Rerunning the phoenix script only changes this...

![image](https://github.com/user-attachments/assets/2c82bb41-68ca-4660-a60e-8984b4e822a7)